App.tsx

```diff
@@ -1,28 +1,56 @@
-import React, { useState } from 'react'
+import React, { Fragment, useCallback, useEffect, useState } from 'react'
 import { StyleSheet, Text, View, Button } from 'react-native'
-import SearchBarScreen from './catalog/SearchBarScreen'
-
+import * as SplashScreen from 'expo-splash-screen'
+import SearchBarScreen from './catalog/GameCardScreen'
+import { useFonts } from 'expo-font'
+import ThemeContextProvider from './components/contexts/ThemeContext'
 
 export default function App() {
-  const [path, setPath] = useState<string | null>(null)
+  const [view, setView] = useState<'screen' | 'root'>('screen')
+  const [fontsLoaded] = useFonts({
+    'Agrandir-Bold': require('./assets/fonts/Agrandir-Bold.otf'),
+    'Agrandir-Medium': require('./assets/fonts/Agrandir-Medium.otf'),
+    'Agrandir-Regular': require('./assets/fonts/Agrandir-Regular.otf'),
+  })
+
+  useEffect(() => {
+    async function prepare() {
+      await SplashScreen.preventAutoHideAsync()
+    }
+    prepare()
+  }, [])
+
+  const onLayoutRootView = useCallback(async () => {
+    if (fontsLoaded) {
+      await SplashScreen.hideAsync()
+    }
+  }, [fontsLoaded])
+
+  if (!fontsLoaded) {
+    return <View />
+  }
+
+  let content = <Fragment />
+
+  if (view === 'root') {
+    content = (
+      <View>
+        <Text style={{ fontFamily: 'Agrandir-Bold' }}>
+          Pick a Screen{'\n\n'}
+        </Text>
+        <Button onPress={() => setView('screen')} title="View Cards" />
+      </View>
+    )
+  } else {
+    content = <SearchBarScreen />
+  }
```

Comment:
@J-Rehan I haven't pulled the code and tried this, but I think App.tsx won't actually work. I think:

* A few lines above, setView should set `gameCard` or something.
* Then you should have a conditional that renders `<GameCardScreen />`